### ISSUE TITLE
Rexec example: Golang server does not work with jsChan client 

### ISSUE THREAD

**Issue by silviucm** [GraftJS/jschan#33]
Hello everyone,

Based on an email conversation I had with Matteo, he suggested I open an issue here, as his assumption was that the Node client should work with the Golang server, based on last time he tested it.

Basically, I am testing the Golang libchan rexec server:
https://github.com/docker/libchan/blob/master/examples/rexec/rexec_server/server.go

Against the Node.Js jsChan rexec sample:
https://github.com/GraftJS/jschan/blob/master/examples/rexec/client.js

The versions tested are the ones in master repo as of January 15, 2015.
I had two console windows open, on the same server. One running a compiled golang version of libchan rexec server, waiting on port 9323.
The other console would be for the client: 
- the golang client run with "rexec echo hi" works correctly
- the node client, run with "node index.js echo hi" does not work correctly  

I added a few log lines in the server, and the approximate location where the server stops being responsive is right after:
t, err := tl.AcceptTransport()  (line 63, it passes this one)
and before 
receiver, err := t.WaitReceiveChannel()  (line 71, it doesn't seem to pass this one, no errors thrown)

In other words, the t.WaitReceiveChannel()  does not appear to be responsive to jsChan's channel.

My regards to the team, great work,
Silviu

**Comment by mcollina** [comment]
This is bad :(. I will try to have a look, but if in the meantime you would like to get it fixed, it might be awesome. I am a little low on bandwidth, so it might take a while :(.

@AdrianRossouw would you like to help tracking this down?

**Comment by AdrianRossouw** [comment]
i don't think keeping in sync with libchan is one of our priorities anymore, since work there seems to have stalled.

we'll probably remove the references to it eventually.
